platform-ui-skin/src/main/webapp/skin/less/social/skin/VersionHistoryDrawer/style.less

```diff
@@ -0,0 +1,99 @@
+/*
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+@import "../../../variables.less";
+
+.VuetifyApp .v-application {
+  .versionHistoryDrawer {
+    .drawerContent {
+      .current_version {
+        background-color: @lightGrey;
+      }
+
+      .item-version-description {
+        width: 100%;
+
+        .descriptionContent {
+          width: 60%;
+          color: @darkGrey !important;
+          font: oblique normal normal 14px/16px Helvetica, sans-serif;
+          position: relative;
+          top: 3px;
+        }
+
+        .description {
+          width: 380px;
+
+          .v-input__append-inner {
+            margin-top: 2px !important;
```

Comment:
is this about a specific style for a generic component? If the style of this component has a bug when integrating it in Product, this directive should go to https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/vuetify/vuetify-all.less

Reply:
It's a specific design, i had to follow the delivered design which is not convenient with default one

Reply:
ok, if not convenient for this generic component switch design, is it for this same component in other UIs? In fact, the idea is to harmonize UIs and UX of all UIs, if a simple outlined `v-text-field` doesn't have a convenient design for all Meeds UIs, we should apply this change everywhere by copying this style in vuetify-all.less to make sure that this component follows our UX Guideline everywhere. (Else we will have this component that will be displayed exceptionally inside this drawer, the way you changed it)
Please, coordinate this with the PO & Designer to make sure that this is the expected UX Design to apply for the product, if not, we shouldn't apply this specific style with a specific UX in this drawer only.

Reply:
@boubaker PR updated